lib/ukstore/README.md

```diff
@@ -0,0 +1,175 @@
+# uk_store
+
+uk_store provides Unikraft libraries with a generic API of getters and setters. The API defines a producer / consumer model that implements pull-oriented getters and push-oriented setters. Under that model, producer libraries define and export a set of entries, each one of which consists of a single value. Consumer libraries retrieve and / or update entries using their associated getters and setter functions.
```

Comment:
One thing I like to do in `md` files is to break paragraphs in one per line. It makes reviewing easier and future additions.

Change only if you want ;)